gamestonk_terminal/portfolio/portfolio_view.py

```diff
@@ -134,7 +134,7 @@ def display_allocation(portfolio: portfolio_model.Portfolio, export: str = ""):
         if gtff.USE_ION:
             plt.ion()
     except Exception as e:
-        print(e)
+        console.print(e)
```

Comment:
If there's an error we don't want to show or export something.

In any case, I don't think we need this try-except block here